src/parser/type_checking.c

```diff
@@ -6797,6 +6797,44 @@ pt_product_sets (PARSER_CONTEXT * parser, TP_DOMAIN * domain, DB_VALUE * set1, D
   return (!pt_has_error (parser));
 }
 
+/*
+ * pt_do_where_type () -
+ *   return:
+ *   parser(in):
+ *   node(in):
+ *   arg(in):
+ *   continue_walk(in):
+ */
```

Comment:
It needs comment to describe the purpose of this routine.

Reply:
It is not related to this issue.